src/qa-components/Main.jsx

```diff
@@ -13,17 +13,18 @@ class QA extends React.Component {
     super(props);
     this.state = {
       questionsAndAnswers: [],
-      helpfulAnswerClicked: false
+      helpfulAnswerClicked: false,
+      helpfulQuestionClicked: false
```

Comment:
I am not sure I understand this correctly, but should each questionsAndAnswers has helpfulAnswerClicked, and helpfulQuestionClicked ? If the Yes is clicked we once the helpfulness go up by one, but if yes is clicked agian we want helpfulness go down by one right ?